README.md

```diff
@@ -449,13 +449,13 @@ object will contain:
 <td>Edge</td>
 
 <!-- Push without payloads support-->
-<td>✗</td>
+<td>✓ 17 (April 2018)</td>
```

Comment:
can you make it v17+ to match other browsers?

Reply:
Done!